toolchains/gcc.env.json

```diff
@@ -1,7 +1,7 @@
 {
   "BAZEL_COMPILER": "gcc",
   "BAZEL_LINKLIBS": "-l%:libstdc++.a",
-  "BAZEL_LINKOPTS": "-lm",
+  "BAZEL_LINKOPTS": "-lm:-fuse-ld=gold",
```

Comment:
not sure if this is correct, but it reverts it to what it was using previously

Reply:
I think this one is correct